common.mk

```diff
@@ -203,6 +245,10 @@ BR2_PACKAGE_OPTEE_TEST_EXT_SITE ?= $(OPTEE_TEST_PATH)
 BR2_PACKAGE_STRACE ?= y
 BR2_TARGET_GENERIC_GETTY_PORT ?= $(if $(CFG_NW_CONSOLE_UART),ttyAMA$(CFG_NW_CONSOLE_UART),ttyAMA0)
 
+ifeq ($(BR2_ROOTFS_POST_BUILD_SCRIPT),$(ROOT)/build/br-ext/board/qemu/post-build.sh)
```

Comment:
I wasn't sure if I can put the args into `qemu_v8.mk` since all the three used variables are defined in `common.mk`, bit it seems to work fine. Also added it to `qemu.mk`.